.eslintrc.json

```diff
@@ -7,7 +7,8 @@
     "Buffer": true,
     "escape": true
   },
-  "extends": "eslint:recommended",
+  "extends": ["eslint:recommended", "prettier"],
```

Comment:
I know this shouldn't be part of this PR since this is the prettier integration but at some point should we use
`https://www.npmjs.com/package/eslint-config-airbnb-base` for the JS linting? this would allow us to start writing some ES6 supported by the newer versions of node without it complaining about things like consts, etc.